nx-dev/ui-cloud/src/lib/faster-and-cheaper.tsx

```diff
@@ -0,0 +1,165 @@
+'use client';
+import {
+  Variants,
+  animate,
+  motion,
+  useMotionValue,
+  useTransform,
+} from 'framer-motion';
+import { useEffect } from 'react';
+import { Spotlight } from './elements/spotlight';
+
+export function FasterAndCheaper(): JSX.Element {
+  const spotlight: Variants = {
+    offscreen: {
+      display: 'none',
+    },
+    onscreen: {
+      display: 'block',
+    },
+  };
+
+  const ciBar: Variants = {
+    start: {
+      width: 0,
+    },
+    end: {
+      width: '100%',
+      transition: {
+        type: 'tween',
+        ease: 'easeOut',
+        duration: 1.2,
+      },
+    },
+  };
+  const nxBar: Variants = {
+    start: {
+      width: 0,
+    },
+    end: {
+      width: '50%',
+      transition: {
+        type: 'tween',
+        ease: 'easeOut',
+        duration: 1.2,
+      },
+    },
+  };
+
+  return (
+    <section>
+      <motion.div
+        initial="offscreen"
+        whileInView="onscreen"
+        viewport={{ once: true }}
+        className="relative mx-auto max-w-7xl px-6 lg:px-8"
+      >
+        <motion.div
+          variants={spotlight}
+          className="absolute left-1/2 z-0 h-[50%] w-[50%] transform-gpu"
+        >
+          <Spotlight className="w-full max-w-full" fill="pink" />
+        </motion.div>
+
+        <div className="mx-auto max-w-2xl text-center">
+          <h2 className="text-3xl font-medium tracking-tight text-slate-950 sm:text-5xl dark:text-white">
+            Both faster & cheaper
+          </h2>
+          <p className="mt-6 text-2xl leading-7 text-slate-700 dark:text-slate-300">
+            Nx Cloud makes your CI significantly faster and cheaper, while also
+            making it more maintainable and reliable.
+          </p>
+        </div>
+        <div className="relative z-20 mx-auto mt-16 max-w-2xl sm:mt-20 lg:mt-24 lg:max-w-5xl">
+          <div className="grid grid-cols-1 gap-x-8 gap-y-16 lg:grid-cols-2">
+            <article className="group/card relative grid transform-gpu cursor-default items-center gap-4 overflow-hidden rounded-xl border border-slate-200 bg-white/50 backdrop-blur-sm transition duration-200 hover:-translate-y-4 hover:shadow-xl md:col-span-1 dark:border-slate-800 dark:bg-slate-950/50 dark:shadow-none">
+              <div className="p-4 text-center">
+                <div className="mt-2 text-3xl font-medium leading-7 text-slate-950 transition duration-200 lg:text-8xl dark:text-white">
+                  <Counter value={30}></Counter> -{' '}
+                  <Counter value={70}></Counter>%
```

Comment:
Can you use the `AnimateValue` component in the `ui-animations` library?
https://github.com/nrwl/nx/blob/master/nx-dev/ui-animations/src/lib/animate-value.tsx